src/FirstSteps/RPG/Heroes/HeroesCreator.cs

```diff
@@ -28,5 +30,29 @@ public static Hero Create(string name, Races race)
                 return null;
             }
         }
+        public static Hero LoadFromFile()
+        {
+
+            HeroModel heroModel = HeroesRepository.LoadHero();
+            Hero hero;
+            Races race = heroModel.Race;
+
+            if (race == Races.Human)
+            {                
+                return hero = new Human(heroModel.Name, heroModel.Damage, heroModel.Health, heroModel.Agility, heroModel.CoinsBag, heroModel.Intelligence, heroModel.Strength);
```

Comment:
`return new Human(heroModel.Name, heroModel.Damage, heroModel.Health, heroModel.Agility, heroModel.CoinsBag, heroModel.Intelligence, heroModel.Strength);` - you don't need `hero` variable, you just have to return a new value;